packages/x-adapter-platform/tsconfig.json

```diff
@@ -2,13 +2,18 @@
   "compilerOptions": {
     "target": "es2019",
     "module": "commonjs",
-    "outDir": "./dist",
+    "outDir": "./dist/cjs",
+    "declaration": true,
+    "declarationDir": "./dist/types",
+    "sourceMap": true,
     "moduleResolution": "node",
     "strict": true,
+    "inlineSources": true,
     "esModuleInterop": true,
-    "types": ["node", "jest"],
-    "lib": ["esnext"]
+    "importHelpers": true,
+    "types": ["jest"],
+    "lib": ["dom", "esnext"]
   },
   "include": ["src/**/*.ts"],
-  "exclude": ["node_modules"]
+  "exclude": ["node_modules", "**/__tests__/**"]
```

Comment:
Excluding the `tests` files will result in `jest` using a different `tsconfig` when running them. This is something we had to refactor in the `x-adapter` package due to miss-matching `typescript` configs between `IDE`, `tests` and `build`.